src/node_native_module.cc

```diff
@@ -36,6 +36,79 @@ using v8::String;
 using v8::Uint8Array;
 using v8::Value;
 
+void NativeModuleLoader::InitializeModuleCategories() {
+  // Already nitialized
```

Comment:
```suggestion
  // Already initialized.
```
(It's a somewhat superfluous comment though.)

Reply:
I originally went with something other than a bool flag..yeah this is not necessary now. I'll remove it.